pytest-embedded/pytest_embedded/plugin.py

```diff
@@ -1273,6 +1297,29 @@ def get_param(item: Function, key: str, default: t.Any = None) -> t.Any:
 
         return item.callspec.params.get(key, default) or default
 
+    @staticmethod
+    def add_app_paths_to_testcases(report_path: str, testcase_app_paths_map: dict, rootdir: str) -> None:
+        with open(report_path, 'r+') as file:
+            tree = ET.parse(file)
+            root = tree.getroot()
+
+            for testcase in root.findall('.//testcase'):
+                pytest_testcase_name = testcase.get('pytest_case_name')
+                if pytest_testcase_name in testcase_app_paths_map:
+                    # Map the app paths as relative paths and add to the 'app_path' attribute
+                    relative_paths = {
+                        os.path.relpath(path, rootdir) for path in testcase_app_paths_map[pytest_testcase_name]
+                    }
+                    testcase.set('app_path', ':'.join(relative_paths))
```

Comment:
```suggestion
                    testcase.set('app_path', ';'.join(relative_paths))
```

better use `;` here. since it's more seldomly used in paths... windows have `:` as drive separator.